inc/notificationtargetcommonitilobject.class.php

```diff
@@ -1550,17 +1550,18 @@ function getDataForObject(CommonDBTM $item, array $options, $simple = false) {
 
             if ($timeline_data['type'] == ITILFollowup::getType()) {
                // Check if the author need to be anonymized
-               if (ITILFollowup::getById($timeline_data['item']['id'])->isFromSupportAgent()
-                  && $is_self_service
+
+               $tmptimelineitem['##timelineitems.author##'] = getUserName($timeline_data['item']['users_id']);
```

Comment:
Should stay in `else`, as it was in original version..

Reply:
refactor done